packages/stream-model-instance/src/model-instance-document.ts

```diff
@@ -320,6 +320,9 @@ export class ModelInstanceDocument<T = Record<string, any>> extends Stream {
     if (!metadata.deterministic) {
       header.unique = randomBytes(12)
     }
+    if (metadata.context) {
```

Comment:
You'll need to add this as an optional field in `ModelInstanceDocumentMetadataArgs`

Reply:
And probably to `GenesisHeader` as well